demo/admin/src/products/ProductsGrid.tsx

```diff
@@ -15,15 +16,18 @@ import {
     ToolbarFillSpace,
     ToolbarItem,
     useBufferedRowCount,
+    useDataGridExcelExport,
     useDataGridRemote,
     usePersistentColumnState,
 } from "@comet/admin";
-import { Add as AddIcon, Edit, StateFilled } from "@comet/admin-icons";
+import { Add as AddIcon, Edit, Excel, MoreVertical, StateFilled } from "@comet/admin-icons";
 import { DamImageBlock } from "@comet/cms-admin";
-import { Button, IconButton, useTheme } from "@mui/material";
+import { Button, CircularProgress, IconButton, Menu, MenuItem, useTheme } from "@mui/material";
 import { DataGridPro, GridFilterInputSingleSelect, GridFilterInputValue, GridToolbarQuickFilter } from "@mui/x-data-grid-pro";
 import gql from "graphql-tag";
-import { FormattedMessage, FormattedNumber, useIntl } from "react-intl";
+import * as React from "react";
+import { useRef, useState } from "react";
```

Comment:
```suggestion
import { useRef, useState } from "react";
```

you already use the named imports, no need for the barrel react import